html/elements/a.json

```diff
@@ -37,6 +37,39 @@
             "deprecated": false
           }
         },
+        "attributionsourceid": {
+          "__compat": {
+            "spec_url": "https://privacycg.github.io/private-click-measurement/#dom-htmlanchorelement-attributionsourceid",
```

Comment:
the html attribute does not has a link to it, so dom property link is used instead